G2P.pm

```diff
@@ -392,6 +395,15 @@ sub new {
     die "The option --filter_by_gene_symbol needs to be set to 1 \n";
   }
   
+  if (defined($params->{only_mane}) and $params->{only_mane} != 1) {
+     $params->{only_mane} = undef;
+    die "The option only_mane needs to be set to 1 \n";
+  }
+  
+  if (defined($params->{only_mane}) and $params->{only_mane} == 1 and $self->{config}->{assembly} ne "GRCh38") {
```

Comment:
At this point, there is no need to check if `$params->{only_mane} == 1`. The job dies at line 400 if only_mane is different than 1.